### PR TITLE
[GC-60] Add usage .env.sample file

### PR DESCRIPTION
**Pull request by Gunn-Wangwichit**
# Description

Add API sample so others can know which API will be used

### What
[//]: # "Briefly describe this feature / fix / refactor (You can write below without deleting this comment)"
- This file is like a template for the environment
### Why
[//]: # "Describe the reason why we need this feature / fix / refactor or refer to Jira Ticket (You can write below without deleting this comment)"
- So people can know which API we are using

Jira Ticket: GC-60
## Type of change

Please check the relevent option with `x`.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if needed)
- [ ] My changes generate no new warnings
- [ ] I have tested on my `Local Machine`

### CODE REVIEW

File: .envsample [hunk 1]
```diff
@@ -0,0 +1,3 @@
+groq_api = "ADD API KEY HERE"
+HF_API_KEY = "ADD API KEY HERE"
```
Comment (Billy19191) [inline .envsample:2]: Please use all env keys as an upper flat case for all. Also, please edit this in code as well :)

**Review by Billy19191 — CHANGES_REQUESTED** [review]

**Comment by Gunn-Wangwichit** [comment]
Pls review again na ja p billy

**Review by mmxxixx — APPROVED** [review]
Good job my man!

**Review by nutkung1 — APPROVED** [review]
good job

**Review by Billy19191 — APPROVED** [review]
LGTM 🫵🏻

**Pull request merged**